api/tests/exceptions/test_exception_handlers.py

```diff
@@ -32,6 +32,16 @@ def push_log_raising():
     assert response.status_code == 401
     assert response.json == {'code': 401, 'message': 'Refresh token error: refresh-token-error'}
 
+def test_validation_error_handler(client, mock_enable_auth, mock_csrf_needed):
```

Comment:
issue: If possible, avoid relying on the actual implementation of `EC2Action`. This way we can safely change the validation of the `ec2_action` endpoint without breaking this test.

I'd recommend just calling the handler and asserting against outputs, given controlled inputs

Reply:
Right! I'll follow your recommendation sir, thanks!